### ISSUE TITLE
centos core dumped

### ISSUE THREAD

**Issue by xiusin** [helix-editor/helix#331]
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ValidationFailed', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/tendril-0.4.2/src/tendril.rs:861:31
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[1]    2026213 abort (core dumped)  ./hx
```

Linux VM-48-10-centos 4.18.0-240.10.1.el8_3.x86_64 #1 SMP Mon Jan 18 17:05:51 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

**Comment by archseer** [comment]
Can you give us a full backtrace via `RUST_BACKTRACE=1`? I'm assuming you tried to open a file that's not UTF-8, this will be addressed by #228

**Comment by archseer** [comment]
Closing due to lack of information, I assume the recently landed encoding support solves this. Let me know if it's still a problem and I'll reopen.
